symplyphysics/laws/thermodynamics/maxwell_boltzmann_distributions/speed_distribution.py

```diff
@@ -0,0 +1,62 @@
+from sympy import Eq, Rational, sqrt, pi, exp
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## For a system containing a large number of identical non-interacting non-relativistic classical
+## particles in thermodynamic equilibrium, the speed distribution function is a function such that
+## `f(v) * dv` gives the fraction of particles with speeds in the interval `dv` at speed `v`.
+
+# Law: f(v) = sqrt(2 / pi) * (m / (k * T))**(3/2) * v**2 * exp(-m * v**2 / (2 * k * T))
+## f(v) - speed distribution function
+## v - speed of particles, i.e. magnitude of velocity vector
+## m - mass of particle
+## k - Boltzmann constant
+## T - equilibrium temperature of the particle ensemble
+
+# Conditions
+## - Number of particles is big enough that the laws of thermodynamics can be applied.
+## - Particles are identical, non-interacting, non-relativistic, and obeying classical laws of physics.
+## - The ensemble of particles is at thermodynamic equilibrium.
+
+speed_distribution_function = Symbol("speed_distribution_function", 1 / units.velocity)
+particle_speed = Symbol("particle_speed", units.velocity)
+particle_mass = Symbol("particle_mass", units.mass)
```

Comment:
We are introducing global symbols. Take a look here: https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/gravity/gravitational_potential_energy.py#L27
And define `particle_mass` with `clone_symbol()`

Reply:
Okay. Why not make it a method, say `.clone(new_name)`, rather than a function?

Reply:
I prefer free functions over class methods. Maybe will refactor this way if we find it more suitable.

Reply:
I see. As far as I saw `sympy` offers both most of the time